generators/client/templates/angular/package.json.ejs

```diff
@@ -174,7 +174,7 @@
     "pretest": "<%= clientPackageManager %> run lint",
     "test": "ng test --coverage --log-heap-usage -w=2",
     "test:watch": "<%= clientPackageManager %> run test -- --watch",
-    "watch": "concurrently npm:start<% if(!skipServer) { %> npm:backend:start<% } %>",
```

Comment:
`npm start` (webpack serve) generates in memory builds.
To write to disk and be accessible at the server port, it needs to execute webpack build instead of serve.
At current setup, gateway forwards to the server port.

Reply:
So it should be active only for microfrontend right? Otherwise there will be a performance impact also for monolith right?

Reply:
I can’t see any problem with performance.
But I’ve already missed the old watch script.
I will duplicate as build-watch.